a1_driver/src/A1_wrapper.cpp

```diff
@@ -6,55 +6,77 @@
 #include "a1_driver/A1_wrapper.h"
 
 void A1Wrapper::walkCmd(float forwardSpeed, float sideSpeed, float rotateSpeed) {
-
-    UNITREE_LEGGED_SDK::HighState current_state;
     UNITREE_LEGGED_SDK::HighCmd high_cmd = {0};
-
-    udp.GetRecv(current_state);
-//    unitree a1 cannot send correct state information now, this function is unfinished. (From Unitree Development Team)
-//    std::cout << unsigned(current_state.mode) << std::endl;
-
-//    step 1 : UDP mode = 1
-    high_cmd.mode = 1;
-    std::cout << "change mode : 0 -> 1, sleep : 2s." << std::endl;
-    udp.SetSend(high_cmd);
-    sleep(2);
-
-//    step 2 : mode = 2
-    high_cmd.mode = 2;
-    std::cout << "change mode : 1 -> 2, sleep : 2s." << std::endl;
+    
+    if (forwardSpeed == 0 && sideSpeed == 0 && rotateSpeed == 0)
```

Comment:
Speed == 0 means the robot is standing or walking in one place, but how to confirm the exact mode?